packages/suite/src/config/suite/sentry.ts

```diff
@@ -3,6 +3,30 @@ import { BrowserOptions } from '@sentry/browser';
 
 export const allowReportTag = 'allowReport';
 
+const beforeSend: BrowserOptions['beforeSend'] = event => {
+    // sentry events are skipped until user confirm analytics reporting
+    const allowReport = event.tags?.[allowReportTag];
+    if (allowReport === false) {
+        return null;
+    }
+    // allow report error without breadcrumbs before confirm status is loaded (@storage/loaded)
+    if (typeof allowReport === 'undefined') {
+        delete event.breadcrumbs;
+    }
+    return event;
+};
+
+const beforeBreadcrumb: BrowserOptions['beforeBreadcrumb'] = breadcrumb => {
+    // filter analytics requests
+    if (
+        breadcrumb.category === 'fetch' &&
+        breadcrumb.data?.url?.contains('data.trezor.io/suite/log')
+    ) {
+        return null;
```

Comment:
Breadcrumb is also full of fetches of images. Maybe we can filter them too?

Reply:
good idea 👍 will do